doc/source/examples/index.rst

```diff
@@ -70,6 +70,15 @@ Mixed Mode
 
     mixedmodeanalysis/*
 
+Vector Fitting
```

Comment:
I don't mind the change, but why is the order important?

Reply:
Well, the Instrument control and the Interactive things are not as well documented as VectorFitting functionalities. So I think it's better to put VectorFitting above the former.